src/bin/cargo-miri.rs

```diff
@@ -347,6 +346,10 @@ path = "lib.rs"
         command.env("RUSTC", find_miri());
     }
     command.env("MIRI_BE_RUSTC", "1");
+    // Make sure there are no other wrappers or flags getting in our way
+    // (Cc https://github.com/rust-lang/miri/issues/1421).
+    command.env_remove("RUSTC_WRAPPER");
+    command.env_remove("RUSTFLAGS");
```

Comment:
Isn't it too invasive? Many flags wouldn't interfere with miri and could be required in certain circumstances.

Reply:
This is preexisting. Also note that this just affects the libstd build, not the build of the local crate. When you do `RUSTFLAGS=foo cargo build`, your libstd also is not affected by the flags you set. So I would say this is consistent with non-Miri operation.